examples/aws-s3-folder/Main.scala

```diff
@@ -61,29 +61,32 @@ val siteDir = "www"
   )
 
   // For each file in the directory, create an S3 object stored in `siteBucket`
-  val uploads: Array[Output[s3.BucketObject]] = File(siteDir).listFiles().map { file =>
+  val uploads: Output[List[s3.BucketObject]] = File(siteDir).listFiles().toList.traverse { file =>
```

Comment:
@lbialy you should be happier with those changes, @KacperFKorban helped me to make it cleaner :)